server/src/graphql/resolvers/mutations.js

```diff
@@ -1,13 +1,14 @@
 const bcrypt = require('bcryptjs');
 const uuidv4 = require('uuid').v4;
 
+const { isValidEmail, isUniqueEmail } = require('../../utils');
 const { db } = require('../../db/models');
-const isValidEmail = require('../../utils/isValidEmail');
 
 module.exports = {
   createAccount: async (_, { model: { email, password } }) => {
     const isValid = isValidEmail(email);
-    if (!isValid) {
+    const isUnique = await isUniqueEmail(email);
+    if (!isValid || !isUnique) {
       return false;
```

Comment:
instead of returning false, you could also return an error message like 'invalid email' or 'email already exist in the database'. returning false for both options doesn't give the FE any info about what went wrong

Reply:
+1